test/integration/lrp/lrp_test.go

```diff
@@ -153,24 +169,43 @@ func TestLRP(t *testing.T) {
 	beforeMetric, err := prometheus.GetMetric(promAddress, coreDNSRequestCountTotal, metricLabels)
 	require.NoError(t, err)
 
-	t.Log("calling nslookup from client")
-	// nslookup to 10.0.0.10 (coredns)
-	val, err := kubernetes.ExecCmdOnPod(ctx, cs, clientDS.Namespace, selectedClientPod, clientContainer, []string{
-		"nslookup", "google.com", "10.0.0.10",
-	}, config)
-	require.NoError(t, err, string(val))
-	// can connect
-	require.Contains(t, string(val), "Server:")
+	t.Log("calling command from client")
+
+	val, errMsg, err := kubernetes.ExecCmdOnPodOnce(ctx, cs, clientPod.Namespace, clientPod.Name, clientContainer, clientCmd, config)
```

Comment:
I think we are not checking this error ?

Reply:
should be here-- I'll reorder it so its right after the exec: https://github.com/Azure/azure-container-networking/pull/3543/files#diff-c3075c56974b82a218bb780da1ac3570cf9421e3e545a1a68afb72eb76083634R180